File: backend/app/api/v1/router.py
from fastapi import APIRouter

from app.api.v1 import assistant, commute, feedback, localities, properties, search

api_router = APIRouter()
api_router.include_router(search.router, prefix="/search", tags=["ai-search"])
api_router.include_router(properties.router, prefix="/properties", tags=["properties"])
api_router.include_router(localities.router, prefix="/localities", tags=["localities"])
api_router.include_router(commute.router, prefix="/commute", tags=["commute"])
api_router.include_router(feedback.router, prefix="/feedback", tags=["feedback"])
api_router.include_router(assistant.router, prefix="/assistant", tags=["assistant"])

File: backend/src/api/v1/router.ts
import { Router } from "express";
import { assistantRouter } from "./assistant.js";
import { commuteRouter } from "./commute.js";
import { communityRouter } from "./community.js";
import { feedbackRouter } from "./feedback.js";
import { localitiesRouter } from "./localities.js";
import { propertiesRouter } from "./properties.js";
import { recommendationsRouter } from "./recommendations.js";
import { searchRouter } from "./search.js";
import { transitRouter } from "./transit.js";

export const apiRouter = Router();

apiRouter.use("/search", searchRouter);
apiRouter.use("/properties", propertiesRouter);
apiRouter.use("/localities", localitiesRouter);
apiRouter.use("/commute", commuteRouter);
apiRouter.use("/feedback", feedbackRouter);
apiRouter.use("/assistant", assistantRouter);
apiRouter.use("/recommendations", recommendationsRouter);
apiRouter.use("/community", communityRouter);
apiRouter.use("/transit", transitRouter);

